pages/Batches/FileFormat.md

```diff
@@ -0,0 +1,63 @@
+---
+menu: Batches
+name: File Format 
+route: /Batches/FileFormat
+---
+
+# Batch Storage
+Batch files are stored and transferred in Apache Parquet format.
+
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Draft |
+
+## Purpose
+1. Specify the format for storing and transferring Batch files.
+1. Specify recommended parameters of the Batch files
+1. Describe reasons for the choice of format.
+
+## Assumptions
+* All assumptions from [DSNP Messages](/Messages/Overview)
+
+## Details
+Parameters for Batch file storage with Bloom filters are be chosen with client applications in mind; consumer devices must be able to query, download and sift through downloaded batch files without adversely affecting user experience by way of
+long download times, heavy processing requirements which may quickly sap battery power, or by using lots of memory.
+
+This has implications for row group size as well as configuring Bloom filters for the batch file, however, since it is possible to announce multiple batches in one announcement, the announcer can announce more data than  is in a single batch file.
+
+## Bloom filter, row group size 
+Optimal settings are still under investigation, however, the maximum row group size allowed in a Parquet file (128M rows) is far too large for a browser or small client application to handle in JavaScript. We are currently defaulting to 128k rows.  
```

Comment:
I think we are still trying for a max one sentence per line. (although long sentences can be split up as well.)
```suggestion
Optimal settings are still under investigation, however, the maximum row group size allowed in a Parquet file (128M rows) is far too large for a browser or small client application to handle in JavaScript.
We are currently defaulting to 128k rows.  
```